source/administration/batch-framework.rst

```diff
@@ -0,0 +1,136 @@
+.. _minio-batch-framework:
+
+===============
+Batch Framework
+===============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. versionchanged:: MinIO RELEASE.2022-10-08T20-11-00Z or later
+
+   The Batch Framework was introduced with the ``replicate`` job type in the :mc:`mc` RELEASES.2022-10-08T20-11-00Z.
+
+Overview
+--------
+
+The MinIO Batch Framework allows you to create, manage, monitor, and execute jobs using a YAML file description.
+Running on the MinIO deployment takes advantage of the cluster's server-side processing power without constraints of the local machine where you run the :ref:`MinIO Client <minio-client>`.
+
+A batch job definition file may contain definitions for a single job task or many job tasks.
+MinIO does not have a pre-defined limit on the number of job tasks that you can define in a definition file.
+
+Initially, the framework allows for creating ``replicate`` jobs to manage bucket replication tasks.
+MinIO anticipates adding additional job types to the framework in future releases.
+
+The MinIO Batch Framework requires that you have :ref:`installed the MinIO Client <mc-install>` and defined the :mc-cmd:`alias <mc alias set>` for the MinIO deployment(s) where you will run the job definition.
+
+The :mc:`mc batch` commands include
+
+.. list-table::
+   :widths: 30 70
+   :width: 90%
+
+   * - :mc:`mc batch generate`
+     - .. include:: /reference/minio-mc/mc-batch-generate.rst
+          :start-after: start-mc-batch-generate-desc
+          :end-before: end-mc-batch-generate-desc
+   * - :mc:`mc batch start`
+     - .. include:: /reference/minio-mc/mc-batch-start.rst
+          :start-after: start-mc-batch-start-desc
+          :end-before: end-mc-batch-start-desc
+   * - :mc:`mc batch list`
+     - .. include:: /reference/minio-mc/mc-batch-list.rst
+          :start-after: start-mc-batch-list-desc
+          :end-before: end-mc-batch-list-desc
+   * - :mc:`mc batch status`
+     - .. include:: /reference/minio-mc/mc-batch-status.rst
+          :start-after: start-mc-batch-status-desc
+          :end-before: end-mc-batch-status-desc
+   * - :mc:`mc batch describe`
+     - .. include:: /reference/minio-mc/mc-batch-describe.rst
+          :start-after: start-mc-batch-describe-desc
+          :end-before: end-mc-batch-describe-desc
+
+Job Types
+---------
+
+Currently, the MinIO Batch Framework supports the following type of job:
+
+- ``replicate``
+
+``replicate``
+~~~~~~~~~~~~~
+
+The ``replicate`` job type performs a single-run replication of objects from one MinIO deployment or location to another MinIO deployment or location.
+The end result of the job is comparable to running :ref:`bucket replication <minio-bucket-replication>`. 
+
+Key advantages of doing replication with a batch job include:
```

Comment:
Since we are comparing this to bucket replication, the below almost implies that bucket replication doesn't do any of these.

I think if we want to compare-contrast this, it's closer to:

- mc batch replicate vs mc mirror - the advantages below
- mc batch replicate vs bucket replication - one time curated vs constant automatic

Reply:
Removed the direct comparison and altered the list of advantages some to.